pcsx2/gui/AppMain.cpp

```diff
@@ -529,10 +531,34 @@ extern bool FMVstarted;
 extern bool renderswitch;
 extern bool EnableFMV;
 
-void DoFmvSwitch()
+void DoFmvSwitch(bool on_off)
```

Comment:
Maybe a different variable name would be better i.e. `on`? `on_off` doesn't really make sense.